cli/flox/src/main.rs

```diff
@@ -79,11 +79,15 @@ fn main() -> ExitCode {
             .unwrap_or_default()
     };
 
+    let disable_metrics = config::Config::parse()
+        .unwrap_or_default()
+        .flox
+        .disable_metrics;
+
+    init_logger(Some(verbosity));
     // Sentry client must be initialized before starting an async runtime or spawning threads
     // https://docs.sentry.io/platforms/rust/#async-main-function
-    let _sentry_guard = init_sentry();
-    init_logger(Some(verbosity));
-
+    let _sentry_guard = init_sentry(disable_metrics);
```

Comment:
personal preference is to conditionally call this rather than passing the bool into that init method, `init_sentry(dont)` is a bit weird imo. besides we have similar construct for our inhouse metrics:

https://github.com/flox/flox/blob/aab753e93ad1633d004cd92210328157f8ffaeec/cli/flox/src/commands/mod.rs#L238-L249


```suggestion
    let _sentry_guard = (!disable_metrics).then(|| init_sentry());
```

Reply:
Done.